charts/access/slack/values.yaml

```diff
@@ -0,0 +1,64 @@
+# Default values for slack.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# replicaCount: 1
+
+image:
+  repository: 146628656107.dkr.ecr.us-west-2.amazonaws.com/gravitational/teleport-plugin-slack
```

Comment:
ECR is the internal. We want `quay.io/gravitational/teleport-plugin-slack` per https://github.com/gravitational/teleport-plugins/pull/489

Reply:
Fixed in https://github.com/gravitational/teleport-plugins/pull/494/commits/a4f6e29685c1ab55a4cfef5d44eda7d1fda9d02e